apps/prairielearn/src/lib/question-render.js

```diff
@@ -137,9 +137,17 @@ async function render(
 
   const studentMessage = 'Error rendering question';
   const courseData = { variant, question, submission, course: variant_course };
+  const user_id = locals.user && locals.user.user_id ? locals.user.user_id : null;
```

Comment:
Nit: consider re-ordering this below the comment on the following line and update the comment to talk about user information in general.

Reply:
Reordered the comment and updated to discuss general user information in [bc258cb](https://github.com/PrairieLearn/PrairieLearn/pull/10840/commits/bc258cb63105129c9382b9367281d7b040a14698).